### DISCUSSION TITLE
I am facing an Issue.

### DISCUSSION THREAD

**Discussion by JayantGoel001** [lowlighter/metrics#285] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/54479676/117196608-525f1f80-ae04-11eb-91b3-c080167a1da8.png)

Hey @lowlighter 
I am getting this issue for 2 days.
Can you please check this out?
Everything was working correctly 2 days before then suddenly this issue came up.

**Comment by lowlighter** [comment, marked as answer]
Hello!
I think this is caused by https://github.com/JayantGoel001/metrics/pull/182
I haven't looked into merging #283 yet on main repo but will eventually do it
If you revert depdendencies to pupetter 9.0.0 on your fork you should be fine I think 🙂

**Reply by JayantGoel001** [reply]
Ya It Totally worked for me
Thanks for the help 😀

**Reply by lowlighter** [reply]
Nice 👍 !
I'll try to check what is preventing puppeteer upgrade 🙂
